app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -17,10 +17,11 @@
 package viewmodels
 
 import controllers.journeys.abroad.routes.{SelfEmploymentAbroadCYAController, SelfEmploymentAbroadController}
+import controllers.journeys.expenses.tailoring.routes.OfficeSuppliesController
 import controllers.journeys.income.routes.{IncomeCYAController, IncomeNotCountedAsTurnoverController}
 import models._
 import models.journeys.Journey
-import models.journeys.Journey.{Abroad, Income}
+import models.journeys.Journey.{Abroad, ExpensesTailoring, Income}
```

Comment:
private val completedStatus      = "completed"
  private val inProgressStatus     = "inProgress"
  private val notStartedStatus     = "notStarted"
  private val cannotStartYetStatus = "cannotStartYet"

These are important statuses, let's give them proper types

```
import enumeratum._

sealed trait TradeJourneyStatus extends EnumEntry {
  override def entryName: String = toString.toLowercase // if you need them lowercased
}

object TradeJourneyStatus extends Enum[TradeJourneyStatus] {
  val values = findValues

  case object Completed extends AccountingType
  case object InProgress    extends AccountingType
...
}
```

Reply:
This will be done in a new PR making richer types in the backend and frontend.